docs/InstallGuide.md

```diff
@@ -131,294 +158,206 @@ To automatically launch the mysql server, execute the following command.
 
     # chkconfig mysqld on
 
-Create Database
+Create database
 
     # mysqladmin -uroot create vnet
     # cd /opt/axsh/openvnet/vnet
     # bundle exec rake db:init
 
+To insert the database records use `vnctl`
+
+```
+# cd /opt/axsh/openvnet/vnctl
+# bundle install --path vendor/bundle
+```
 
-Start the OpenVNet services
----------------------------
+Start vnmgr and webapi since `vnctl` is just a webapi client.
```

Comment:
This comment about vnctl should be in the last section where we introduced the reader to vnctl.